frontend/src/metabase/browse/components/ModelsTable.styled.tsx

```diff
@@ -9,8 +9,8 @@ import type { ResponsiveProps } from "metabase/components/ItemsTable/utils";
 import { breakpoints } from "metabase/ui/theme";
 
 export const ModelTableRow = styled.tr<{ skeleton?: boolean }>`
-  :outline {
-    outline: 2px solid var(--mb-color-brand);
+  :focus {
+    outline: 2px solid var(--mb-color-focus);
   }
```

Comment:
Is the `--mb-color-focus` change expected?

Reply:
yes, it's made by the code author